src/fastmcp/mcp_config.py

```diff
@@ -104,7 +105,11 @@ def to_transport(self) -> FastMCPTransport:
             exclude_tags=self.exclude_tags,
         )
 
-        return FastMCPTransport(wrapped_mcp_server)
+        return wrapped_mcp_server, transport
```

Comment:
Is this wrapped_mcp_server used? could we get away with just returning the transport as to_transport()?

Oh I see below where it's used -- maybe rather than have the conditional you could just create the server there, and always hit `to_transport()`? Just a thought

Reply:
this is a bug actually, `to_transport` is supposed to wrap the server in a FastMCPTransport and return it, not directly return the underlying transport. Will update quick

Reply:
if we just return the normal transport than it wont be a transforming server, itll just be a normal server